pkg/types/libvirt/machinepool.go

```diff
@@ -3,11 +3,20 @@ package libvirt
 // MachinePool stores the configuration for a machine pool installed
 // on libvirt.
 type MachinePool struct {
+	DomainMemory int32 `json:"memory"`
```

Comment:
why int32?

make sure you have field docs like https://github.com/openshift/installer/blob/05388774c311c8dbf791c702cbd7678fd163739e/pkg/types/libvirt/platform.go#L6

Reply:
also maybe name it `DomainMemoryMiB` ?

Reply:
all done. Moved to plain `int`